jina/flow/__init__.py

```diff
@@ -790,6 +790,56 @@ def my_reader():
         """
         self._get_client(**kwargs).search(input_fn, output_fn, **kwargs)
 
+    def plot(self, output='', copy_flow: bool = True) -> 'Flow':
```

Comment:
```suggestion
    def plot(self, output: str=None, copy_flow: bool = False) -> 'Flow':
```

also make sure the docstring is added for parameters